server/src/main/java/org/bonitasoft/console/common/server/auth/impl/standard/StandardAuthenticationManagerImpl.java

```diff
@@ -48,13 +45,22 @@ public String getLoginPageURL(final HttpServletRequestAccessor request, final St
             context += "/mobile";
         }
         url.append(context).append(AuthenticationManager.LOGIN_PAGE).append("?");
-        if (request.getTenantId() != null) {
-            url.append(AuthenticationManager.TENANT).append("=").append(request.getTenantId()).append("&");
+        String tenantId = getTenantId(request);
+        if (tenantId != null) {
+            url.append(AuthenticationManager.TENANT).append("=").append(tenantId).append("&");
         }
         url.append(AuthenticationManager.REDIRECT_URL).append("=").append(redirectURL);
         return url.toString();
     }
 
+    private String getTenantId(HttpServletRequestAccessor request) {
+        String tenantId = request.getTenantId();
+        if (tenantId == null) {
+            tenantId = PortalCookies.getTenantCookieFromRequest(request.asHttpServletRequest());
+        }
+        return tenantId;
+    }
+
```

Comment:
Yep I've changed only the StandardAuthenticationManager

AFAIK there is only CASRemoteAuthenticationManagerImpl in bonita-web-sp, I'll check with @redboul if we need to change it also